### PR TITLE
Buttontheme

### PR DESCRIPTION
**Pull request by lkttyagi**
Issue Browse button coming out of segment (issue #606) fixed.

### CODE REVIEW

**Comment by lkttyagi** [comment]
is there any way to generate changelog automatically & how to resolve this conflict in changelog ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Change Log
+##1.6.3 (2019-03-03)
```
Comment (tisto) [inline CHANGELOG.md]: @lkttyagi this messed up the changelog. Please fix.

File: theme/themes/default/globals/site.variables [hunk 2]
```diff
@@ -76,8 +76,8 @@
 
 /* This adjusts the default form input across all elements */
 @inputBackground        : @white;
-@inputVerticalPadding   : @relative11px;
-@inputHorizontalPadding : @relative14px;
+@inputVerticalPadding   : @relative1px;
```
Comment (tisto) [inline theme/themes/default/globals/site.variables]: @lkttyagi are you sure this does not affect any other css styles in the site? You are changing some base settings here, not just a single css statement.

File: theme/themes/default/elements/button.variables [hunk 3]
```diff
@@ -18,6 +18,7 @@
 @verticalPadding: @inputVerticalPadding;
 @horizontalPadding: 1.5em;
 
+
```
Comment (tisto) [inline theme/themes/default/elements/button.variables]: @lkttyagi please try to not add blank lines in files when you don't change anything. I'd recommend to install prettier and some HTML/CSS cleaner add ons. What editor are you using?

**Comment by tisto** [comment]
@lkttyagi you have to fix the changelog conflict manually I am afraid. Please also check the Travis build and run "yarn run test" locally to see the test failures.

File: theme/themes/default/globals/site.variables (hunk 2 above)
Comment (lkttyagi) [reply theme/themes/default/globals/site.variables]: yes  it doesnt change any other css styles.

File: theme/themes/default/elements/button.variables (hunk 3 above)
Comment (lkttyagi) [reply theme/themes/default/elements/button.variables]: sure . i am using sublime text

**Comment by lkttyagi** [comment]
@tisto i change the changelog and check all test on localhost and they all are passed but here both plone and guillotina test are failed.

**Comment by nileshgulia1** [comment]
@lkttyagi try to merge from master.

**Pull request closed without merging**